docs/src/components/DocsBehaviorRoot.tsx

```diff
@@ -18,7 +18,7 @@ class DocsBehaviorRoot extends React.Component<any, any> {
   }
 
   baseName(fileName: string) {
-    const divided = _.startCase(fileName.replace('ts', ''))
+    const divided = _.startCase(fileName.replace(/\.ts$/, ''))
     return _.upperFirst(_.lowerCase(divided))
```

Comment:
May be we can move this to `gulp-component-menu-behaviors` and call this prop as `title`?

Reply:
actually, would like just to introduce fix first - but generally absolutely agree, let me consider this move for the follow up PR :)